src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_x509.h

```diff
@@ -42,7 +42,7 @@ Returns 1 on success, 0 on failure, any other value on invalid state.
 
 Output:
 pPublicKeyOut: Receives a CFRetain()ed SecKeyRef for the public key
-pOSStatusOut: Receives the result of SecCertificateCopyPublicKey
+pOSStatusOut: Receives the result of SecCertificateCopyKey
```

Comment:
Receives the result of SecCertificateCopyKey or SecCertificateCopyPublicKey, depending on the OS version.